### PR TITLE
falco-edits

### PR DESCRIPTION
**Pull request by ankitasharma88990**
may grammaratically errors are corrected.

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> /kind flaky-test

> If contributing rules or changes to rules, please make sure to also uncomment one of the following line:

> /kind rule-update

> /kind rule-create

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area engine

> /area examples

> /area rules

> /area integrations

> /area tests

> /area proposals

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by ankitasharma88990** [comment]
Please accept as it's the end of hacktoberfest.
Thank you.

File: CHANGELOG.md [hunk 1]
```diff
@@ -59,9 +59,9 @@ Released 2019-07-12
 
 ## Major Changes
 
-* Clean up error reporting to provide more meaningful error messages along with context when loading rules files. When run with -V, the results of the validation ("OK" or error message) are sent to standard output. [[#708](https://github.com/falcosecurity/falco/pull/708)]
+* Clean up error reporting to provide more meaningful error messages along with context when loading rules files. When running with -V, the results of the validation ("OK" or error message) are sent to standard output. [[#708](https://github.com/falcosecurity/falco/pull/708)]
```
Comment (leodido) [inline CHANGELOG.md]: Not sure here calling help from native English speakers (@kris-nova, @mstemm)

File: CHANGELOG.md [hunk 2]
```diff
@@ -99,13 +99,13 @@ Released 2019-07-12
 
 * Allow k8s.gcr.io/kube-proxy image to run privileged. [[#717](https://github.com/falcosecurity/falco/pull/717)]
 
-* Add runc to the list of possible container entrypoint parents. [[#712](https://github.com/falcosecurity/falco/pull/712)]
+* Add runs to the list of possible container entry point parents. [[#712](https://github.com/falcosecurity/falco/pull/712)]
```
Comment (leodido) [inline CHANGELOG.md]: `runc` is correct here.

Also not sure about entry point versus entrypoint

File: CHANGELOG.md [hunk 3]
```diff
@@ -99,13 +99,13 @@ Released 2019-07-12
...
 
 * Skip Source RFC 1918 addresses when considering outbound connections. [[#685](https://github.com/falcosecurity/falco/pull/685)]
 
 * Add additional `user_XXX` placeholder macros to allow for easy customization of rule exceptions. [[#685](https://github.com/falcosecurity/falco/pull/685)]
 
-* Let weaveworks programs change namespaces. [[#685](https://github.com/falcosecurity/falco/pull/685)]
+* Let weave works programs change namespaces. [[#685](https://github.com/falcosecurity/falco/pull/685)]
```
Comment (leodido) [inline CHANGELOG.md]: `weaveworks` was correct here

File: CHANGELOG.md [hunk 4]
```diff
@@ -117,7 +117,7 @@ Released 2019-07-12
 
 * Add back trusted_containers list for backport compatibility [[#675](https://github.com/falcosecurity/falco/pull/675)]
 
-* Add mkdirat as a syscall for mkdir operations. [[#667](https://github.com/falcosecurity/falco/pull/667)]
+* Add moderate as a syscall for media operations. [[#667](https://github.com/falcosecurity/falco/pull/667)]
```
Comment (leodido) [inline CHANGELOG.md]: `mkdirat` correct, `mkdir` too

File: CHANGELOG.md [hunk 5]
```diff
@@ -219,15 +219,15 @@ Released 2019-05-13
...
 
-* Move Kubernetes Response Engine to own repo: The Kubernetes Response Engine is now in its [own github repository](https://github.com/falcosecurity/kubernetes-response-engine). [[#539](https://github.com/falcosecurity/falco/pull/539)]
+* Move Kubernetes Response Engine to own repo: The Kubernetes Response Engine is now in its [own GitHub repository](https://github.com/falcosecurity/kubernetes-response-engine). [[#539](https://github.com/falcosecurity/falco/pull/539)]
 
 * Updated Puppet Module: An all-new puppet module compatible with puppet 4 with a smoother installation process and updated package links. [[#537](https://github.com/falcosecurity/falco/pull/537)] [[#543](https://github.com/falcosecurity/falco/pull/543)] [[#546](https://github.com/falcosecurity/falco/pull/546)]
 
-* RHEL-based falco image: Provide dockerfiles that use RHEL 7 as the base image instead of debian:unstable. [[#544](https://github.com/falcosecurity/falco/pull/544)]
+* RHEL-based Falco image: Provide docker files that use RHEL 7 as the base image instead of debian: unstable. [[#544](https://github.com/falcosecurity/falco/pull/544)]
```
Comment (leodido) [inline CHANGELOG.md]: `dockerfiles` is correct

File: CHANGELOG.md [hunk 6]
```diff
@@ -270,7 +270,7 @@ Released 2019-05-13
 
 * Better coverage for cassandra/kubelet/kops to reduce FPs [[#551](https://github.com/falcosecurity/falco/pull/551)]
 
-* Better coverage for docker, openscap to reduce FPs [[#573](https://github.com/falcosecurity/falco/pull/573)]
+* Better coverage for docker, open cap to reduce FPs [[#573](https://github.com/falcosecurity/falco/pull/573)]
```
Comment (leodido) [inline CHANGELOG.md]: `openscap` correct here

File: CHANGELOG.md [hunk 7]
```diff
@@ -282,13 +282,13 @@ Released 2019-02-06
...
 
 * Allow SSL for K8s audit endpoint/embedded webserver [[#471](https://github.com/falcosecurity/falco/pull/471)]
 
-* Add stale issues bot that automatically flags old github issues as stale after 60 days of inactivity and closes issues after 67 days of inactivity. [[#500](https://github.com/falcosecurity/falco/pull/500)]
+* Add stale issues bot that automatically flags old GitHub issues as stale after 60 days of inactivity and closes issues after 67 days of inactivity. [[#500](https://github.com/falcosecurity/falco/pull/500)]
 
-* Support bundle: When run with `--support`, falco will print a json object containing necessary information like falco version, command line, operating system information, and falco rules files contents. This could be useful when reporting issues. [[#517](https://github.com/falcosecurity/falco/pull/517)]
+* Support bundle: When running with `--support`, Falco will print a json object containing necessary information like falco version, command line, operating system information, and falco rules files contents. This could be useful when reporting issues. [[#517](https://github.com/falcosecurity/falco/pull/517)]
```
Comment (leodido) [inline CHANGELOG.md]: Same as above (@kris-nova, @mstemm ?). Run or running?

File: CHANGELOG.md [hunk 8]
```diff
@@ -364,15 +364,15 @@ Released 2018-11-09
 
 ## Major Changes
 
-* **Support for K8s Audit Events** : Falco now supports [K8s Audit Events](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-backends) as a second stream of events in addition to syscalls. For full details on the feature, see the [wiki](https://github.com/falcosecurity/falco/wiki/K8s-Audit-Event-Support).
+* **Support for K8s Audit Events **: Falco now supports [K8s Audit Events](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-backends) as a second stream of events in addition to syscalls. For full details on the feature, see the [wiki](https://github.com/falcosecurity/falco/wiki/K8s-Audit-Event-Support).
```
Comment (leodido) [inline CHANGELOG.md]: Why adding a white space here?

File: COPYING [hunk 9]
```diff
@@ -14,11 +14,11 @@
       the copyright owner that is granting the License.
 
       "Legal Entity" shall mean the union of the acting entity and all
-      other entities that control, are controlled by, or are under common
```
Comment (koppor) [inline COPYING]: Did you notice that this file is the [Apache License](https://en.wikipedia.org/wiki/Apache_License). Did you check with a laywer that this is still the same license? If the license text is changed, the license needs to be approved again by the "open source initiative".

Comment (leodido) [reply COPYING]: Agree with @koppor , this changes MUST be reverted too.

**Comment by leodido** [comment]
Going to close this since it is huge and it seems to been done with automatic tools not accounting for specific situations.

/close

**Pull request closed without merging**